app/controllers/exam_templates_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ExamTemplatesController < ApplicationController
+
+  before_filter      :authorize_only_for_admin
+
+  layout 'assignment_content'
+
+  def index
+    @assignment = Assignment.find(params[:assignment_id])
+    @exam_templates = ExamTemplate.all
+  end
+
+  def download
+    assignment = Assignment.find(params[:assignment_id])
+    exam_template = ExamTemplate.find_by(assignment: assignment)
+    filename = exam_template.filename
+    basename = File.basename(filename, ".pdf")
+    send_file("#{Rails.root}/data/dev/exam_templates/#{basename}/#{filename}",
```

Comment:
I believe there's an option in the config files (check `development.rb`) that specifies the path. You should look into `markus_config_*` methods to see how we use them to access configuration options.